### PR TITLE
feat: lists and text utilities

### PR DESCRIPTION
**Pull request by tushartiwari7**
## Lists
list

## Headings
head

### CODE REVIEW

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.li {
+  list-style: none;
+}
```
Comment (JantuDeb) [inline components/list/list.css:3]: You can use a better name for the class here.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

File: components/components.css [hunk 2]
```diff
@@ -5,6 +5,10 @@
 @import './card/card.css';
 @import './image/image.css';
 @import './input/input.css';
+@import './list/list.css';
+
+@import './textUtility/textUtility.css';
+
```
Comment (harshitadaswani) [inline components/components.css:11]: remove extra space

File: layout/sidebar.css [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
   grid-auto-rows:max-content;
   transition: .25s ease-in;
+  overflow: scroll;
 }
 
+.sidebar::-webkit-scrollbar {
+  width: 5px;
+}
+
+
```
Comment (harshitadaswani) [inline layout/sidebar.css:13]: remove empty spaces

**Review by harshitadaswani — COMMENTED** [review]
just remove empty spaces and merge it.

**Review by harshitadaswani — APPROVED** [review]
You can go ahead and merge it.

**Pull request merged**